src/extensions/styles/migrators/buttonIconTransitionMigrator.js

```
@@ -1,27 +1,34 @@
+	if (!transition?.block?.icon) return newAttributes;
+
+	const { block } = transition;
+	const iconData = block.icon;
+
+	// Use for loop for better performance
+	for (let i = 0; i < NEW_KEYS.length; i++) {
+		block[NEW_KEYS[i]] = iconData;
 	}
 
+	// Direct deletion instead of creating new object
+	delete block.icon;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)